.github/workflows/testing.yml

```diff
@@ -8,6 +8,8 @@ permissions:
 on:
   pull_request:
 
+permissions: {}
```

Comment:
There already is a `permissions` entry a few lines above.

Reply:
Thanks!